github/src/main/java/com/novoda/github/reports/service/network/GithubApiService.java

```diff
@@ -71,6 +72,7 @@
             @Query("per_page") int perPageCount
     );
 
+    @Deprecated
```

Comment:
Sorry, but I am not familiar with the reason for this deprecation. Is there another (better) way to get the timeline?